### ISSUE TITLE
LockActivity is blank on Android 5.0.1 Lollipop

### ISSUE THREAD

**Issue by ritdaw** [auth0/Lock.Android#49]
I'm using the sample android application, the authentication activity (LockActvity) is launching successfully but the the activity is blank when only using the Username-Password-Authentication provider. Initially you see the Auth0 logo but it disappears. If I enable a social provider such as G+ the login fields appear.

**Comment by hzalaz** [comment]
@ritdaw I don't have an HTC One at hand to try it. But It's working with all the devices I have with Lollipop. 
By any chance do you see an error in the logs? and also what are the connections enabled for the app, just DB connection or more?

**Comment by ritdaw** [comment]
Hi,

After further testing it appears

**Comment by ritdaw** [comment]
Hi,

Just username and password enabled. If I enable G+ as well all the controls display correctly.

**Comment by hzalaz** [comment]
Are you seeing any error in the console?. I have this working in several devices with lollipop but I don't have a HTC One to try it. It might be something wrong with the HTC or with your account.
Can you also send to support's email your tenant and clientId?

**Comment by ritdaw** [comment]
I've amended the title slightly as the issue isn't related to a specific device and perhaps not the version of Android either. If I enable G+ and Username & password authentication all works fine. If I turn off G+ the activity is blank (after loading it briefly shows the Auth0 icon and a spinner). See the logcat output, note I have removed the JSON output for security reasons.

03-19 07:47:55.397  20130-20130/com.sample.mobileservices V/com.auth0.lock.LockActivity﹕ Resuming activity with data null
03-19 07:47:55.397  20130-20130/com.sample.mobileservices V/com.auth0.api.APIClient﹕ Fetching application info from https://cdn.auth0.com/client/8wm6LvGoIzheF6FKOn0bmlb2Vms9qnQ9.js
03-19 07:47:55.471      505-533/? I/ActivityManager﹕ Displayed com.sample.mobileservices/com.auth0.lock.LockActivity: +125ms
03-19 07:47:55.486  20130-20153/com.sample.mobileservices D/OpenGLRenderer﹕ endAllStagingAnimators on 0x55765613c0 (RippleDrawable) with handle 0x55767a1c60
03-19 07:47:55.578  20130-20130/com.sample.mobileservices V/AsyncHttpResponseHandler﹕ Progress 630 from 1 (63000%)
03-19 07:47:55.583  20130-20130/com.sample.mobileservices D/com.auth0.api.handler.ApplicationResponseHandler﹕ Obtained JSON object from JSONP: {data removed for security]
03-19 07:47:55.713  20130-20130/com.sample.mobileservices D/com.auth0.api.APIClient﹕ Obtained application with id 8wm6LvGoIzheF6FKOn0bmlb2Vms9qnQ9 tenant auth0534
03-19 07:47:55.713  20130-20130/com.sample.mobileservices I/com.auth0.lock.fragment.LoadingFragment﹕ Fetched app info for tenant auth0534
03-19 07:47:55.713  20130-20130/com.sample.mobileservices D/com.auth0.lock.LockActivity﹕ Application configuration loaded for id 8wm6LvGoIzheF6FKOn0bmlb2Vms9qnQ9
